frontend/src/stories/components/Courses/CoursesTable.stories.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```

Comment:
Since there's a new story, please add a link to that story in the PR description.

To find the story, navigate to the website for the repo:

<img width="492" alt="image" src="https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-1/assets/1119017/9b6923a4-fb58-45c3-aee1-423c29517d2d">


Then find the PR: 

<img width="545" alt="image" src="https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-1/assets/1119017/cd7158de-1668-4898-9d21-39e9856a56f7">

Follow the link to the storybook:

<img width="251" alt="image" src="https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-1/assets/1119017/79734ba8-37a5-4d67-9c82-5e2cda1beb18">

The follow the link to the new story: 

<img width="244" alt="image" src="https://github.com/ucsb-cs156-f23/proj-organic-f23-6pm-1/assets/1119017/2533d5d1-ffb1-458e-be68-2913f6c9ec5a">

Grab that link, and that's the link you want to put into the PR description.


https://ucsb-cs156-f23.github.io/proj-organic-f23-6pm-1/prs/48/storybook/?path=/story/components-courses-coursestable--three-items-ordinary-user